packages/widget-notes/src/components/FeatureInterestDialog.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button, DialogActions, Typography } from '@mui/material'
```

Comment:
If we use the same dialog for notes, snippets and todos let's also just have one component for all and move it into `marquee-utils`.